internal/server/data/data.go

```diff
@@ -64,6 +65,14 @@ func NewSQLiteDriver(connection string) (gorm.Dialector, error) {
 			return nil, err
 		}
 	}
+	uri, err := url.Parse(connection)
+	if err != nil {
+		return nil, err
+	}
+	query := uri.Query()
+	query.Add("_journal_mode", "WAL")
```

Comment:
Should we be checking if the uri already contains this query?

Reply:
only if someone would ever want to disable WAL. Not sure that we should give them the choice.